Testing/SystemTests/tests/qt/SliceViewerIntegrationTest.py

```diff
@@ -506,12 +580,27 @@ def runTest(self):
         # set nonorthog view and retrieve new limits
         pres.nonorthogonal_axes(True)
         limits_nonorthog = pres.view.data_view.get_data_limits_to_fill_current_axes()
-        self.assertAlmostEqual(limits_nonorthog[0][0], -19, delta=1e-5)
-        self.assertAlmostEqual(limits_nonorthog[0][1], 19, delta=1e-5)
+        self.assertAlmostEqual(limits_nonorthog[0][0], -10, delta=1e-5)
```

Comment:
It changed because I didn't understand the expected behavior to fill the entire data beyond the parallelogram region into the rectangle. I need to switch it back.